test/ejabberd_tests/tests/muc_http_api_SUITE.erl

```diff
@@ -0,0 +1,285 @@
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%% http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%
+%% Author: Joseph Yiasemides <joseph.yiasemides@erlang-solutions.com>
+%% Description: Test HTTP Administration API for Mult-user Chat (MUC)
+%%==============================================================================
+
+-module(muc_http_api_SUITE).
+-compile(export_all).
+
+-include_lib("escalus/include/escalus.hrl").
+-include_lib("escalus/include/escalus_xmlns.hrl").
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("exml/include/exml.hrl").
+
+
+%%--------------------------------------------------------------------
+%% Suite configuration
+%%--------------------------------------------------------------------
+
+all() ->
+    [{group, positive}].
+
+groups() ->
+    [{positive, [parallel], success_response() ++ complex()}].
+
+success_response() ->
+    [
+     create_room,
+     invite_online_user_to_room,
+     %% invite_offline_user_to_room, %% TO DO.
+     send_message_to_room
+    ].
+
+complex() ->
+    [
+     multiparty_multiprotocol
+    ].
+
+
+%%--------------------------------------------------------------------
+%% Init & teardown
+%%--------------------------------------------------------------------
+
+init_per_suite(Config) ->
+    muc_helper:load_muc(muc_helper:muc_host()),
+    escalus:init_per_suite(Config).
+
+end_per_suite(Config) ->
+    muc_helper:unload_muc(),
+    escalus_fresh:clean(),
+    escalus:end_per_suite(Config).
+
+init_per_group(_GroupName, Config) ->
+    escalus:create_users(Config, escalus:get_users([alice, bob, kate])).
+
+end_per_group(_GroupName, Config) ->
+    escalus:delete_users(Config, escalus:get_users([alice, bob, kate])).
+
+init_per_testcase(CaseName, Config0) ->
+    Config1 = [{room_name, make_distinct_name(<<"wonderland">>)}|Config0],
+    escalus:init_per_testcase(CaseName, Config1).
+
+end_per_testcase(CaseName, Config) ->
+    muc_helper:destroy_room(muc_helper:muc_host(), ?config(room_name, Config)),
+    escalus:end_per_testcase(CaseName, Config).
+
+
+%%--------------------------------------------------------------------
+%% Tests
+%%--------------------------------------------------------------------
+
+create_room(Config) ->
+    escalus:fresh_story(Config, [{alice, 1}], fun(Alice) ->
+        Host = <<"localhost">>,
+        Path = <<"/mucs/", Host/binary>>,
+        Name = ?config(room_name, Config),
+        Body = #{name => Name,
+                 owner => escalus_utils:get_jid(Alice),
```

Comment:
I missed that before. All the jids used in `Path` or `Body` should be bare jids. Could you please change it here and in other places in this suite?